src/SFA.DAS.EmployerIncentives.UnitTests/Application/Services/EmployerIncentivesService/WhenApiIsUnavailable.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Threading.Tasks;
+using FluentAssertions;
+using NUnit.Framework;
+
+namespace SFA.DAS.EmployerIncentives.UnitTests.Application.Services.EmployerIncentivesService
+{
+    [TestFixture] 
+    public class WhenApiIsUnavailable
+    {
+        [Test]
+        public async Task Then_health_check_should_return_false()
```

Comment:
The title for this should be Pascal case - this test is already covered in `WhenCallingIsHealthy` in `SFA.DAS.EmployerIncentives.UnitTests.Application.EligibleApprenticeships.Services` and the test for when it returns not healthy

Reply:
fixed case. Same comment.